staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service.go

```diff
@@ -54,7 +54,7 @@ type gRPCService struct {
 
 // NewGRPCService returns an envelope.Service which use gRPC to communicate the remote KMS provider.
 func NewGRPCService(endpoint string, callTimeout time.Duration) (Service, error) {
-	klog.V(4).Infof("Configure KMS provider with endpoint: %s", endpoint)
+	klog.V(4).Infof("Configure KMS provider with testSocket: %s", endpoint)
```

Comment:
bad find/replace of endpoint with testSocket in messages/comments?

Reply:
yes.
Fixed.